compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposePanel.desktop.kt

```diff
@@ -42,13 +42,21 @@ import org.jetbrains.skiko.SkiaLayerAnalytics
  * @param skiaLayerAnalytics Analytics that helps to know more about SkiaLayer behaviour.
  * SkiaLayer is underlying class used internally to draw Compose content.
  * Implementation usually uses third-party solution to send info to some centralized analytics gatherer.
+ * @param useSwingGraphics Flag indicating whether to use Swing graphics for rendering.
+ * Defaults to the "compose.swing.render.on.graphics" environment value.
  * @param renderSettings Configuration class for rendering settings.
  */
 class ComposePanel @ExperimentalComposeUiApi constructor(
     private val skiaLayerAnalytics: SkiaLayerAnalytics,
+    @property:ExperimentalComposeUiApi
+    private val useSwingGraphics: Boolean = ComposeFeatureFlags.useSwingGraphics,
```

Comment:
Maybe it should go into `RenderSettings` below?

Reply:
Probably no.
1. It will be a parameter not only for `ComposePanel` that I'm trying to avoid in this PR
2. I see a place where it's used only if `useSwingGraphics == false`

Reply:
If we look at the API from user perspective, it might be better to have these parameters under 1 thing, just from `ComposePanel` user perspective.

The points are valid, but we can fix them by creating something like:
```
sealed class ComposePanelRenderSettings {
  class LightweightLayer: ComposePanelRenderSettings()
  class HeavyweightLayer(val renderSettings: RenderSettings): ComposePanelRenderSettings 
}
```

Lightweight, heavyweight - Swing terms.

I think it is better to figure out the shape now rather than trying to avoid big changes. Because we will stuck with it forever, and it looks not consistent to have 2 params that affect rendering separately.